UITextView+Placeholder/UITextView+Placeholder.m

```diff
@@ -144,16 +145,18 @@ - (void)updatePlaceholderLabel {
     }
 
     [self insertSubview:self.placeholderLabel atIndex:0];
+    [self.placeholderLabel sizeToFit];
 
     self.placeholderLabel.font = self.font;
     self.placeholderLabel.textAlignment = self.textAlignment;
-    [self.placeholderLabel sizeToFit];
 
     CGRect frame = self.placeholderLabel.frame;
     frame.origin.x = self.textContainer.lineFragmentPadding + self.textContainerInset.left;
     frame.origin.y = self.textContainerInset.top;
     frame.size.width = (CGRectGetWidth(self.bounds) - frame.origin.x -
                         self.textContainer.lineFragmentPadding - self.textContainerInset.right);
+    frame.size.height = [self.placeholderLabel sizeThatFits:frame.size].height;
```

Comment:
At line 148 and 158, `sizeThatFits` called twice. I think we can use `sizeToFit` just once at the end of setting `placeholderLabel.font` and `textAlignment`. (original code)

Is this intended?